src/QMCHamiltonians/ACForce.h

```diff
@@ -23,69 +23,71 @@
 
 namespace qmcplusplus
 {
-struct ACForce : public OperatorBase
+class ACForce : public OperatorBase
 {
-  typedef ParticleSet::ParticlePos_t Force_t;
+public:
+  using Force_t = ParticleSet::ParticlePos_t;
```

Comment:
Don't see any reason for the _t  Force is obviously a type.

Reply:
Additionally its actually the vector of the per particle forces.  so `Forces` would provide a much better clue for what is going on.